frontend/public/components/image-stream.tsx

```diff
@@ -203,24 +238,28 @@ ImageStreamsPage.displayName = 'ImageStreamsListPage';
 
 /*  eslint-disable no-undef, no-unused-vars  */
 type ImageStreamTagsRowProps = {
-  imageStream: any,
-  specTag: any,
-  statusTag: any,
+  imageStream: K8sResourceKind;
+  specTag: any;
+  statusTag: any;
+};
+
+export type ImageStreamManipulationHelpProps = {
+  imageStream: K8sResourceKind;
 };
 
 export type ImageStreamsRowProps = {
-  obj: any,
+  obj: any;
```

Comment:
Existing code, but this really should be

```suggestion
  obj: K8sResourceKind;
```

here and just below